### PR TITLE
Create and add secundary pages

### PR DESCRIPTION
**Pull request by delhomvi**
Cette PR va ajouter les Features suivantes:
- Création des pages "secondaires" 
- Lien avec les cartes/liens/boutons de la main page
- MAJ du front en conséquence

### CODE REVIEW

**Comment by delhomvi** [comment]
no merge needed for now, still missing router link of tiles

**Comment by delhomvi** [comment]
review can be started
there is still better front needed, but i will open another PR for this, every links/cards component are linked to another component, there is still some minor issue like for exemple link a children component as a parent component, but for now i'll works with this

File: Angular/WebApp-FilRouge/src/app/header/hamburger-menu/hamburger-menu.component.html [hunk 1]
```diff
@@ -10,6 +10,10 @@
       <li class="searchbar"><app-searchbar></app-searchbar></li>
       <li class="buttons"><app-buttons></app-buttons></li>
       <li><a routerLink="home">Home</a></li>
+      <li><a routerLink="liste">Liste des formations</a></li>
+      <li><a routerLink="create">Demander la création d'une formation</a></li>
+      <li><a routerLink="search">Rechercher une formation</a></li>
```
Comment (taos-thr) [inline Angular/WebApp-FilRouge/src/app/header/hamburger-menu/hamburger-menu.component.html:15]: "rechercher une formation" dont fit in the burger menu

Comment (taos-thr) [reply Angular/WebApp-FilRouge/src/app/header/hamburger-menu/hamburger-menu.component.html:15]: I try the commit [200c009] and the issue is resolved !

**Review by taos-thr — APPROVED** [review]
Everything works !

**Pull request merged**